### DISCUSSION TITLE
Integrate a chatbot with NLP/NLU capabilities into Amadeus GDS

### DISCUSSION THREAD

**Discussion by enstobbi** [community/community#78883] [Programming Help]
I'm looking for someone who can help me integrate a chatbot with NLP/NLU capabilities into Amadeus. I would like to create a chatbot that emulates the work of a Travel Agent using Amadeus - <Staff removed personal info>

**Comment by samus-aran** [comment]
Hi @enstobbi , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! 

We want to make sure you’re getting the best support you can now that GitHub Universe has ended so I've gone ahead and moved this to the most relevant category for you.

Thank you for helping us maintain a productive and tidy community for all our members.
